### PR TITLE
Update extensions table.

### PR DESCRIPTION
**Pull request by dgash**
Add lines from actual extensions list; make punctuation/capitalization consistent; delete amp-slides (not supported).

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -16,23 +16,31 @@ include the following script in the `<head>`:
...
-| [`amp-instagram`](amp-instagram/amp-instagram.md)           | Displays an instagram embed.                                                                |
-| [`amp-install-serviceworker`](amp-install-serviceworker/amp-install-serviceworker.md)               | Installs a ServiceWorker.
-| [`amp-lightbox`](amp-lightbox/amp-lightbox.md)             | Allows for a “lightbox” or similar experience.                                              |
-| [`amp-list`](amp-list/amp-list.md)             | A dynamic list that can download data and create list items using a template |
-| [`amp-twitter`](amp-twitter/amp-twitter.md)               | Displays a Twitter Tweet.                                                                   |
-| [`amp-vine`](amp-vine/amp-vine.md)               | Displays a Vine simple embed.                                                                   |
-| [`amp-youtube`](amp-youtube/amp-youtube.md)               | Displays a Youtube video.                                                                   |
+| Component | Description |
+| --------- | ----------- |
+| [`amp-access`](amp-access/amp-access.md) | Provides AMP paywall and subscription support.  |
```
Comment (Meggin) [inline extensions/README.md:21]: This should be amp-access-spec and amp-access-spec.md

File: extensions/README.md [hunk 2]
```diff
@@ -16,23 +16,31 @@ include the following script in the `<head>`:
...
+| [`amp-analytics`](amp-analytics/amp-analytics.md) | Capture analytics data from an AMP document. |
```
Comment (Meggin) [inline extensions/README.md:22]: Small nit-- some descriptions use plural for verb, some don't. Might be nice to make these consistent.

**Comment by Meggin** [comment]
Hey, Dave.

Noticed just a few small nits. If these are easy to fix from your side, then cool. I can also merge the PR, and fix on my end. See if it's easy to edit again and add to the same PR. I think it should be.

**Comment by dgash** [comment]
I dunno... when I go into edit, I get the original version of the file
before my changes. ?????

**Comment by Meggin** [comment]
Cool. I'll merge and then you can fix them in the merged version :).

**Comment by Meggin** [comment]
Done.

**Comment by dgash** [comment]
Okay I'm seeing the new one now. Will make those changes and resubmit.
Thanks!

**Comment by dgash** [comment]
Made the changes, submitted pull request. Thanks; that looks better.

**Pull request merged**